From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Tue, 5 Jan 2016 03:43:19 +0100
Subject: crm_resource: avoid printing uninitialized memory

---
 tools/crm_resource.c |   16 +++++++---------
 1 file changed, 7 insertions(+), 9 deletions(-)

diff --git a/tools/crm_resource.c b/tools/crm_resource.c
index 1b2976b..1d35747 100644
--- a/tools/crm_resource.c
+++ b/tools/crm_resource.c
@@ -375,18 +375,16 @@ main(int argc, char **argv)
                 } else if (safe_str_eq("list-agents", longname)) {
                     lrmd_list_t *list = NULL;
                     lrmd_list_t *iter = NULL;
-                    char standard[512];
-                    char provider[512];
+                    char *provider = index (optarg, ':');
                     lrmd_t *lrmd_conn = lrmd_api_new();
 
-                    rc = sscanf(optarg, "%[^:]:%s", standard, provider);
-                    if (rc == 1) {
+                    if (!provider) {
                         rc = lrmd_conn->cmds->list_agents(lrmd_conn, &list, optarg, NULL);
-                        provider[0] = '*';
-                        provider[1] = 0;
+                        provider = "*";
 
-                    } else if (rc == 2) {
-                        rc = lrmd_conn->cmds->list_agents(lrmd_conn, &list, standard, provider);
+                    } else {
+                        *provider++ = 0;
+                        rc = lrmd_conn->cmds->list_agents(lrmd_conn, &list, optarg, provider);
                     }
 
                     if (rc > 0) {
@@ -398,7 +396,7 @@ main(int argc, char **argv)
                         lrmd_list_freeall(list);
                         rc = 0;
                     } else {
-                        fprintf(stderr, "No agents found for standard=%s, provider=%s\n", standard,
+                        fprintf(stderr, "No agents found for standard=%s, provider=%s\n", optarg,
                                 provider);
                         rc = -1;
                     }
