From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 13 Sep 2015 19:59:05 +0200
Subject: Extend the description of the remote agent script
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Ferenc Wágner <wferi@niif.hu>
---
 extra/resources/remote |   10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/extra/resources/remote b/extra/resources/remote
index 447267e..b6cb36f 100644
--- a/extra/resources/remote
+++ b/extra/resources/remote
@@ -44,8 +44,14 @@ meta_data() {
 <?xml version="1.0"?>
 <!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
 <resource-agent name="remote" version="0.1">
-<version>0.1</version>
-<shortdesc lang="en">remote resource agent</shortdesc>
+  <version>0.1</version>
+  <shortdesc lang="en">provide metadata for the remote connection agent</shortdesc>
+  <longdesc lang="en">
+This resource agent script only provides the metadata of the internal
+remote lrmd connection agent to announce its parameters.  All other
+operations are unsupported.  The actual functionality lives within the
+crmd component of pacemaker.
+  </longdesc>
   <parameters>
     <parameter name="server" unique="1">
     <longdesc lang="en">
