From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 13 Sep 2015 20:43:33 +0200
Subject: Fix typo: maintainence -> maintenance

---
 ChangeLog       |    2 +-
 pengine/graph.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ChangeLog b/ChangeLog
index e445890..a604303 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -820,7 +820,7 @@
   + mcp: Set the HA_quorum_type env variable consistently with our corosync plugin
   + mcp: Shut down if one of our child processes can/should not be respawned
   + pengine: Bug cl#5000 - Ensure ordering is preserved when depending on partial sets
-  + pengine: Bug cl#5028 - Unmanaged services should block shutdown unless in maintainence mode
+  + pengine: Bug cl#5028 - Unmanaged services should block shutdown unless in maintenance mode
   + pengine: Bug cl#5038 - Prevent restart of anonymous clones when clone-max decreases
   + pengine: Bug cl#5007 - Fixes use of colocation constraints with multi-state resources
   + pengine: Bug cl#5014 - Prevent asymmetrical order constraints from causing resource stops
diff --git a/pengine/graph.c b/pengine/graph.c
index 197fa7f..0688590 100644
--- a/pengine/graph.c
+++ b/pengine/graph.c
@@ -629,7 +629,7 @@ shutdown_constraints(node_t * node, action_t * shutdown_op, pe_working_set_t * d
         } else if (action->node->details != node->details) {
             continue;
         } else if (is_set(action->rsc->flags, pe_rsc_maintenance)) {
-            pe_rsc_trace(action->rsc, "Skipping %s: maintainence mode", action->uuid);
+            pe_rsc_trace(action->rsc, "Skipping %s: maintenance mode", action->uuid);
             continue;
         } else if (node->details->maintenance) {
             pe_rsc_trace(action->rsc, "Skipping %s: node %s is in maintenance mode",
