From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 13 Sep 2015 20:26:22 +0200
Subject: Library options go into Libs, not Cflags

---
 lib/pacemaker.pc.in |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/pacemaker.pc.in b/lib/pacemaker.pc.in
index 0e69ff5..071e431 100644
--- a/lib/pacemaker.pc.in
+++ b/lib/pacemaker.pc.in
@@ -31,6 +31,6 @@ Version:      @PACKAGE_VERSION@
 Description:  Pacemaker cluster manager v@PACKAGE_SERIES@
 Requires:     
 Conflicts:    
-Cflags:       -I${includedir} -lcrmcommon
-Libs:         -L${libdir} 
+Cflags:       -I${includedir}
+Libs:         -L${libdir} -lcrmcommon
 Libs.private:
