From: =?utf-8?q?Ferenc_W=C3=A1gner?= <wferi@niif.hu>
Date: Sun, 13 Sep 2015 19:36:57 +0200
Subject: Substitute additional configure macros used in pkg-config files

---
 configure.ac |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/configure.ac b/configure.ac
index e361e29..4734af4 100644
--- a/configure.ac
+++ b/configure.ac
@@ -1196,6 +1196,7 @@ fi
 
 AC_DEFINE_UNQUOTED(SUPPORT_UPSTART, $HAVE_upstart, Support upstart based system services)
 AM_CONDITIONAL(BUILD_UPSTART, test $HAVE_upstart = 1)
+AC_SUBST(SUPPORT_UPSTART)
 
 if
     $PKGCONFIG --exists systemd
@@ -1215,6 +1216,7 @@ fi
 
 AC_DEFINE_UNQUOTED(SUPPORT_SYSTEMD, $HAVE_systemd, Support systemd based system services)
 AM_CONDITIONAL(BUILD_SYSTEMD, test $HAVE_systemd = 1)
+AC_SUBST(SUPPORT_SYSTEMD)
 
 case $SUPPORT_NAGIOS in
      1|yes|true|try)
@@ -1393,6 +1395,7 @@ AC_DEFINE_UNQUOTED(HAVE_ATOMIC_ATTRD, $BUILD_ATOMIC_ATTRD, Support the new atomi
 
 AC_SUBST(SUPPORT_CMAN)
 AC_SUBST(SUPPORT_CS)
+AC_SUBST(SUPPORT_PLUGIN)
 
 dnl
 dnl    Cluster stack - Sanity
